app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+file_name = input("Enter name of the file:")
+
+with open(f"{file_name}.txt", "w") as f:
+    line = ""
+    while line != "stop":
+        line = input("Enter new line of content:")
+        f.write(f"{line}\n")
```

Comment:
Unfortunaltely, your solution doesn't work correctly, because when your input is "stop" it will be written in the last line of your file. 
You can look on the screanshot:
https://prnt.sc/n7aiuTdVpPWB